src/components/vuestic-components/va-divider/VaDivider.demo.vue

```diff
@@ -0,0 +1,88 @@
+<template>
```

Comment:
Couple of important things missing:
* context support (I added support, but we need to demo)
* documentation